C#/cs01/tests/11.cs

```diff
@@ -0,0 +1,42 @@
+using NUnit.Framework;
+using Brainchild.Curriculum.Solutions;
+namespace Brainchild.Curriculum
+{
+    [TestFixture]
+    public class Test11
+    {
+        [SetUp]
+        public void SetUp()
+        {
+        }
+        
+        [Test]
+        public void CheckingFullNaming()
+        {
+            Sreerag sreerag = new Sreerag("Sreerag P S");
```

Comment:
why are you passing fullname , you should pass firstname ,middlename and lastname